### DISCUSSION TITLE
Bootstrap 3.4.1 vulnerabilities  - do they exist?

### DISCUSSION THREAD

**Discussion by c-schmitz** [twbs/bootstrap#36001] [Q&A] [answered]
I have a hard time figuring out if there are any vulnerabilities in the last Bootstrap 3 version, namely 3.4.1
I see that there are vulnerabilities reported for later 4.x versions, but it is not exactly clear if they also exist in 3.x.

We are still using Boostrap 3 and want to have an impression how urgent an upgrade is to be recommended.
Thank you for any insight,

Carsten

**Comment by coliff** [comment, marked as answer]
Bootstrap **3.4.1** is fine (No known vulnerabilities found).
https://snyk.io/test/npm/bootstrap/3.4.1

**3.4.0** has a medium severity vulnerability.
https://snyk.io/test/npm/bootstrap/3.4.0

Hope that helps!

**Reply by ashutoshvimal** [reply]
Hi @coliff can you confirm is there any new vulneribilty now ? As I am not able to see any vulnerabilities.

**Reply by coliff** [reply]
This page displays the vulnerabilities for each version of Bootstrap:
https://security.snyk.io/package/npm/bootstrap
There are no known issues since v5.
